fil_actors_shared/build.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use regex::Regex;
+use std::collections::HashSet;
+use std::path::{Path, PathBuf};
+use std::{env, fs};
+use walkdir::WalkDir;
+
+fn main() {
+    let manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let actors_dir = manifest_dir.join("../actors");
+    if env::var("SKIP_ACTOR_VERSION_CHECK").is_ok() || !actors_dir.exists() {
```

Comment:
1. Would it be easier to opt-in for the actor version check? We could have it in the CI and Makefile, but anything outside of it would be 100% unaffected.
2. Let's have some common prefix, e.g., `FIL_...`

Reply:
Done